.github/workflows/release.yml

```diff
@@ -58,6 +64,7 @@ jobs:
           HUSKY: 0
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NPM_CONFIG_REGISTRY: ${{ inputs.releaseToNpm == true && 'https://example.com' || 'https://npm.petrov.ca/' }}
```

Comment:
I purposefully put example.com. I want to make sure its releasing to the petrov domain and if its successful on beta branches ill add npm registry here.

Basically the thought is, the workflow dispatch (meaning we just go to github and invoke the workflow and pass the input) means we can invoke the npm deployment (like it was automated on push, just requires us to actually push the button).